plugins/postcss-cascade-layers/test/wpt/layer-basic.html

```diff
@@ -0,0 +1,548 @@
+									@layer {
+											target { color: red; }
+									}
+							}
+							@layer A {
+									@layer {
+											target { color: green; }
+									}
+							}
+					`,
+				},
+				{
+					title: 'D4 Mixed named and anonymous layers',
+					style: `
+							@layer A {
+									@layer {
+											target { color: red; }
+									}
+							}
+							@layer {
+									target { color: green; }
+							}
+							@layer A {
+									@layer {
+											target { color: red; }
+									}
+							}
+					`,
+				},
+				{
+					title: 'D5 Mixed named and anonymous layers',
+					style: `
+							@layer {
+									@layer A {
+											target { color: red; }
+									}
+							}
+							@layer {
+									target { color: green; }
+							}
+					`,
+				},
+				{
+					title: 'E1 Statement syntax',
+					style: `
+							@layer A, B, C;
+							@layer A {
+									target.first { color: red; }
+									target.second { color: red; }
+							}
+							@layer B {
+									target.first { color: red; }
+							}
+							@layer C {
+									target.first { color: green; }
+									target.second { color: green; }
+							}
+					`,
+				},
+				{
+					title: 'E2 Statement syntax',
+					style: `
+							@layer A, C, B;
+							@layer A {
+									target.first { color: red; }
+									target.second { color: red; }
+							}
+							@layer B {
+									target.first { color: green; }
+							}
+							@layer C {
+									target.first { color: red; }
+									target.second { color: green; }
+							}
+					`,
+				},
+				{
+					title: 'E3 Statement syntax',
+					style: `
+							@layer C, B, A;
+							@layer A {
+									target.first { color: green; }
+									target.second { color: green; }
+							}
+							@layer B {
+									target.first { color: red; }
+							}
+							@layer C {
+									target.first { color: red; }
+									target.second { color: red; }
+							}
+					`,
+				},
+				{
+					title: 'E4 Statement syntax',
+					style: `
+							@layer B, A.B, A.A;
+							@layer A {
+									@layer A {
+											target.first { color: green; }
+									}
+									@layer B {
+											target.first { color: red; }
+											target.second { color: green; }
+									}
+							}
+							@layer B {
+									target { color: red; }
+							}
+					`,
+				},
+				{
+					title: 'E5 Statement syntax',
+					style: `
+							@layer A.B, B, A.A;
+							@layer A {
+									@layer A {
+											target.first { color: red; }
+									}
+									@layer B {
+											target.first { color: red; }
+											target.second { color: red; }
+									}
+							}
+							@layer B {
+									target { color: green; }
+							}
+					`,
+				},
+			];
+
+			for (let testCase of testCases) {
+				const styleElement = document.createElement('style');
+
+				const testCaseStyle = await (fetch(`/test/styles.css`, {
+					method: 'POST',
+					body: testCase['style'],
+				}).then((response) => response.text()));
```

Comment:
To preserve the WPT tests as much as possible the test case contents is send via `fetch` to the test server and transformed by the plugin on the fly.

This setup makes it easy to later add more cases if WPT is updated.